src/components/transport_manager/include/transport_manager/transport_adapter/transport_adapter_impl.h

```diff
@@ -91,7 +91,7 @@ class TransportAdapterImpl : public TransportAdapter,
    * @return true if initialized.
    * @return false if not initialized.
    */
-  virtual bool IsInitialised() const;
+  virtual bool IsInitialised() const OVERRIDE;
```

Comment:
@vlantonov , @anosach-luxoft meant that if you added `OVERRIDE` - `virtual` can be deleted as only virtual methods can be added by `OVERRIDE`. Therefore `OVERRIDE` automatically means that function is `virtual` but not on the contrary.